README.md

```diff
@@ -21,7 +22,7 @@ Follow the [instructions](https://code.visualstudio.com/docs/editor/extension-ga
 
 ## Known Issues
 
-* when you create a new file and add the `/* @flow */` to the beginning you need to manually change the mode from the status bar to `Flow`
+* You should set workspace preference to disabe default syntax validation from Visual Studio Code.
```

Comment:
Typo in "disable".

Also, can you be a little bit more specific about how to do this? In the PR comments you said you need to set:

`"javascript.validate.syntaxValidation": true`

I think it's worth putting that down explicitly.